#
# Ubuntu: https://bugs.launchpad.net/ubuntu/+source/mp3blaster/+bug/452231
# Description: Fix a FTBFS when converting from 'const char*' to 'char*'.
#
diff -Nur -x '*.orig' -x '*~' mp3blaster/src/global.cc mp3blaster.new/src/global.cc
--- mp3blaster/src/global.cc	2009-10-15 17:03:20.124589776 +0200
+++ mp3blaster.new/src/global.cc	2009-10-15 17:04:27.830949852 +0200
@@ -379,7 +379,7 @@
 is_sid(const char *filename)
 {
 #ifdef HAVE_SIDPLAYER
-	char *ext = strrchr(filename, '.');
+	const char *ext = strrchr(filename, '.');
 	if (ext) {
 		if (!strcasecmp(ext, ".psid")) return 1;
 		if (!strcasecmp(ext, ".sid")) return 1;
